src/signals/incident-management/containers/IncidentOverviewPage/index.test.js

```diff
@@ -83,38 +73,84 @@ describe('signals/incident-management/containers/IncidentOverviewPage', () => {
     };
   });
 
-  it('should render correctly', () => {
-    const { queryByTestId, rerender, getByText } = render(
+  it('should render modal buttons', () => {
+    const { getByText } = render(
       withAppContext(<IncidentOverviewPageContainerComponent {...props} />)
     );
 
-    expect(queryByTestId('pagination')).not.toBeNull();
-    expect(queryByTestId('incidentOverviewListComponent')).not.toBeNull();
-    expect(queryByTestId('loadingIndicator')).toBeNull();
+    expect(getByText('Filteren').tagName).toEqual('BUTTON');
+    expect(getByText('Mijn filters').tagName).toEqual('BUTTON');
+  });
 
-    const loadingProps = cloneDeep(props);
-    loadingProps.overviewpage.loading = true;
+  it('should render a list of incidents', () => {
+    const { queryByTestId, rerender } = render(
+      withAppContext(<IncidentOverviewPageContainerComponent {...props} />)
+    );
+
+    expect(queryByTestId('loadingIndicator')).toBeInTheDocument();
+    expect(
+      queryByTestId('incidentOverviewListComponent')
+    ).not.toBeInTheDocument();
+
+    const incidents = generateIncidents();
 
     rerender(
       withAppContext(
-        <IncidentOverviewPageContainerComponent {...loadingProps} />
+        <IncidentOverviewPageContainerComponent
+          {...props}
+          incidents={{
+            count: incidents.length,
+            results: incidents,
+            loading: false,
+          }}
+        />
       )
     );
 
-    expect(queryByTestId('incidentOverviewPagerComponent')).toBeNull();
-    expect(queryByTestId('incidentOverviewListComponent')).toBeNull();
-    expect(queryByTestId('loadingIndicator')).not.toBeNull();
+    expect(queryByTestId('loadingIndicator')).not.toBeInTheDocument();
+    expect(queryByTestId('incidentOverviewListComponent')).toBeInTheDocument();
+  });
+
+  it('should render pagination controls', () => {
+    constants.FILTER_PAGE_SIZE = 101;
+    const incidents = generateIncidents();
 
-    // filter button
-    expect(getByText('Filteren').tagName).toEqual('BUTTON');
-    expect(getByText('Mijn filters').tagName).toEqual('BUTTON');
+    const { queryByTestId, rerender } = render(
+      withAppContext(
+        <IncidentOverviewPageContainerComponent
+          {...props}
+          incidents={{
+            count: incidents.length,
+            results: incidents,
+            loading: false,
+          }}
+        />
+      )
+    );
+
+    expect(queryByTestId('pagination')).not.toBeInTheDocument();
+
+    constants.FILTER_PAGE_SIZE = 99;
```

Comment:
And what about when `FILTER_PAGE_SIZE === incidentCount`?

Reply:
Adde a test for it